src/resolve/resolvectl.c

```diff
@@ -1302,19 +1302,38 @@ static int map_link_domains(sd_bus *bus, const char *member, sd_bus_message *m,
 }
 
 static int status_print_strv_ifindex(int ifindex, const char *ifname, char **p) {
+        const unsigned indent = strlen("Global: "); /* Use the same indentation everywhere to make things nice */
+        int pos1, pos2;
+
+        if (ifname)
+                printf("%s%nLink %i (%s)%n%s:", ansi_highlight(), &pos1, ifindex, ifname, &pos2, ansi_normal());
+        else
+                printf("%s%nGlobal%n%s:", ansi_highlight(), &pos1, &pos2, ansi_normal());
+
+        unsigned cols = columns(), position = pos2 - pos1 + 2;
         char **i;
 
-        printf("%sLink %i (%s)%s:",
-               ansi_highlight(), ifindex, ifname, ansi_normal());
+        STRV_FOREACH(i, p) {
+                unsigned our_len = strlen(*i);
```

Comment:
doesn't matter much, but maybe use utf8_console_width()? after all hostnames can be literal utf-8 on mdns, and even with idna dns too